src/components/KCollapse/KCollapse.vue

```diff
@@ -111,6 +121,25 @@ export default defineComponent({
         return ['leading', 'trailing'].includes(value)
       },
     },
+    type: {
+      type: String as PropType<'default' | 'tags'>,
+      required: false,
+      default: 'default',
+    },
+    tags: {
+      type: Array,
+      default: () => [],
+    },
+    tagsAppearance: {
+      type: String as PropType<'default' | 'light'>,
+      required: false,
+      default: 'default',
+    },
```

Comment:
The tag appearance should be integrated into the `tags` prop if it's going to be a thing; there's no need for a separate prop. `tags` can be an array of objects, similar to:
```ts
[
  {
    appearance: '',
    text: '',
  }
]
```